src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+
+  if (days < 3) {
+    return days * costPerDay;
+  } else if (days >= 3 && days < 7) {
+    return days * costPerDay - 20;
+  } else {
+    return days * costPerDay - 50;
+  }
```

Comment:
if you immediately return do not add unnecessary `else if`. just two `if`s would be cleaner

Reply:
also, when looking at this code i have no idea what 20 and 50 are. it would be good to create variables to signify their meaning